lib/charms/glauth_k8s/v0/ldap.py

```diff
@@ -320,28 +322,38 @@ def _on_relation_broken(self, event: RelationBrokenEvent) -> None:
         )
         secret.remove()
 
+    def get_app_password(self, relation_id: int) -> Optional[str]:
+        """Retrieve the secret for a given relation."""
```

Comment:
Just some minor things:

- Is `get_bind_password` a better name?
- docstring: `Retrieve the bind account password for a given integration.`?
- Probably the return type can be unified to `str` if we return `secret.get_content().get("password", "")`? or we return `None` when we handle the `SecretNotFoundError`.

Reply:
+1 on the first 2 points

Reply:
+1

Reply:
Fix it